src/include/kernel/ext_manifest.h

```diff
@@ -73,6 +73,21 @@ enum ext_man_elem_type {
 	EXT_MAN_ELEM_FW_VERSION		= 0,
 	EXT_MAN_ELEM_CC_VERSION		= SOF_IPC_EXT_CC_INFO,
 	EXT_MAN_ELEM_PROBE_INFO		= SOF_IPC_EXT_PROBE_INFO,
+	EXT_MAN_ELEM_CONFIG_DATA	= 5,
+};
+
+/* EXT_MAN_ELEM_CONFIG_DATA elements identificators */
+enum config_elem_type {
+	EXT_MAN_CONFIG_CAVS_LPRO	= 1,
+	EXT_MAN_CONFIG_IPC_MSG_SIZE,
+	EXT_MAN_CONFIG_OUTBOX_SIZE,
+	EXT_MAN_CONFIG_INBOX_SIZE,
+	EXT_MAN_CONFIG_LAST_ELEM,	/**< keep it at the end of enum list */
+};
+
+struct config_elem {
+    uint32_t token;
+    uint32_t value;
 };
```

Comment:
doesn't this structure have to be packed? and same question to `const struct ext_man_config_data ext_man_config` below

Reply:
both of members are uint32_t so no issues but actually yes, it should, I will update